scripts/runmal.py

```diff
@@ -61,7 +64,7 @@ def cleanup():
     conn.close()
 
     # All done, write the stamp
-    stampfile = os.path.join(logdir, 'stamps', run_id)
+    stampfile = os.path.join(logdir, 'stamps', sample_name)
```

Comment:
What's the idea behind these changes? IIRC, `run_id` was intended to be a UUID, whereas `sample_name` was the filename of the sample. I had been trying to use `run_id` everywhere because we may want to run the same sample more than once. (But I may be misremembering the purpose of these variables)

Reply:
Right for the purpose of dumping the memory and finding the process in the list of dumped vads later on having the name was pretty important. One solution here would be to add a cli argument for naming scheme type so that either are available. What do you think?

Reply:
Hmm, right now the UUID to filename maps is put into an sqlite database so it can be retrieved later for things like matching the VADs. Would that work here?